csm_web/scheduler/tests/utils.py

```diff
@@ -1,71 +0,0 @@
-from os import path
-from django.test import TestCase
-from rest_framework.test import APIClient
```

Comment:
This file should probably be kept around for now, as it's not relevant to the PR (we'll likely make another issue and PR removing these past tests).